lib/pf/Authentication/Source/KickboxSource.pm

```diff
@@ -0,0 +1,120 @@
+package pf::Authentication::Source::KickboxSource;
+
+=head1 NAME
+
+pf::Authentication::Source::NullSource add documentation
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::Authentication::Source::NullSource
```

Comment:
What a great name ! But hey, that's the description field ! How about a description instead ? ;)